api/main.go

```diff
@@ -61,6 +61,22 @@ func run(ctx context.Context, store handler.Store) error {
 	}
 }
 
+func getAddress() string {
+	var host, port string
+	var exists bool
+
+	host, exists = os.LookupEnv("HOST")
+	if !exists {
+		host = "localhost"
```

Comment:
I think the default should be `0.0.0.0`

Reply:
Good point @sansaid ! Thank you for pointing this out :)